python/pip_install/pip_repository.bzl

```diff
@@ -550,7 +550,7 @@ of a binary found on the host's `PATH` environment variable. If no value is set
 If you are using a custom python interpreter built by another repository rule,
 use this attribute to specify its BUILD target. This allows pip_repository to invoke
 pip using the same interpreter as your toolchain. If set, takes precedence over
-python_interpreter.
+python_interpreter. An example value: "@python3_x86_64-unknown-linux-gnu//:bin/python3".
```

Comment:
nit: Just `//:python` can be used. All our python_repository rules create a symlink for that name pointing to the bin/python3 file.

Reply:
TODO

Reply:
Done

Reply:
I still see :bin/python3 here?